### DISCUSSION TITLE
MIMIC-IV vs MIMIC-IV-ED

### DISCUSSION THREAD

**Discussion by mdbotto** [MIT-LCP/mimic-code#1560] [MIMIC-IV]
Hello,
I'm sorry if this is a silly question, but I am not a medical doctor and I'm still trying to figure everything out about MIMIC.
In MIMIC-IV, there are patients who were admitted either into the Emergency Department (ED) or an ICU between 2008 and 2019.
In MIMIC-IV-ED, the data are patients who were admitted into the ED between 2011 and 2019.

I have a couple of questions:

- What is the need for MIMIC-IV-ED? Isn't every patient admitted to the ED in MIMIC-IV already?
- Are scheduled doctor appointments of any kind in BIDMC included in MIMIC-IV?

**Comment by alistairewj** [comment]
> What is the need for MIMIC-IV-ED? Isn't every patient admitted to the ED in MIMIC-IV already?

Yes, but there isn't any information *from* the ED information system. So for small subset of earlier patients, you know they went through the ED, but you don't know their triage vitals, medication reconciliation, and so on.

> Are scheduled doctor appointments of any kind in BIDMC included in MIMIC-IV?

Hmmm by and large no. Just those who go through the ED or ICU. You could imagine an elective admission resulting in an ICU stay however, so there may be a few.
